valhalla/thor/worker.h

```diff
@@ -6,6 +6,9 @@
 #include <vector>
 
 #include <boost/property_tree/ptree.hpp>
+#ifdef ENABLE_GDAL
+#include <gdal_priv.h>
+#endif
```

Comment:
it'd be great if we wouldn't leak gdal into our headers. it's easy to push this particular one into its .cc file, not sure about the serializer though. the reason is: if it's in our public headers, then a consuming application also needs the gdal headers, while it'd only need a gdal library if it's in our .cc files (i.e. only in our static/shared library)